src/site/xdoc/server/config-blob.xml

```diff
@@ -20,84 +20,31 @@
-                      Default is cassandra.
-                  </dd>
-              </dl>
-          </p>
-
-          <subsection name="Swift BlobStore Configuration">
-              <dl>
-                  <dt><strong>objectstorage.payload.codec</strong></dt>
-                  <dd>AES256: Aes256 encrypted blob content. DEFAULT: non encrypted</dd>
-
-                  <dt><strong>objectstorage.provider</strong></dt>
-                  <dd>Should be swift</dd>
-
-                  <dt><strong>objectstorage.namespace</strong></dt>
-                  <dd>Namespace to store blobs in</dd>
-
-                  <dt><strong>objectstorage.swift.authapi</strong></dt>
-                  <dd>Should be tmpauth</dd>
-
-                  <dt><strong>objectstorage.swift.endpoint</strong></dt>
-                  <dd>Swift authentication endpoint. Should be http://swift-host-address:8080/auth/v1.0 </dd>
-
-                  <dt><strong>objectstorage.swift.credentials</strong></dt>
-                  <dd>Should be testing</dd>
-
-                  <dt><strong>objectstorage.swift.tempauth.username</strong></dt>
-                  <dd>Should be tester</dd>
-
-                  <dt><strong>objectstorage.swift.tempauth.tenantname</strong></dt>
-                  <dd>Should be test</dd>
-
-                  <dt><strong>objectstorage.swift.tempauth.passheadername</strong></dt>
-                  <dd>Should be X-Storage-Pass</dd>
-
-                  <dt><strong>objectstorage.swift.tempauth.passheadername</strong></dt>
-                  <dd>Should be X-Storage-User</dd>
-              </dl>
-          </subsection>
-      </subsection>
-
       <subsection name="Blob Export Configuration">
           <p>
-              <a href="https://github.com/apache/james-project/blob/master/server/blob/blob-export-api/src/main/java/org/apache/james/blob/export/api/BlobExportMechanism.java">BlobExportMechanism</a>
-              is a component used to export blob content to a target email address. It requires to has a BlobId, an string of explanation, and target mail address in order to perform exporting.
+              Configuration for exporting blob content
           </p>
           <dl>
               <dt><strong>blob.export.implementation</strong></dt>
-              <dd>localFile: Local File Exporting Mechanism. Default: localFile</dd>
+              <dd>localFile: Local File Exporting Mechanism (explained below). Default: localFile</dd>
           </dl>
 
           <subsection name="Local File Blob Export Configuration">
               <p>
-                  This mechanism export blob data to a local zip file, then send an emails contains the absolute path of that file to the target mail address
+                  This mechanism export blob data to a local zip file, then send an email containing the absolute path of that file to the target mail address.
```

Comment:
is it true? do we build a zip file with a single blob in it?

Reply:
I replace that sentence by this:
```
For each request, this mechanism retrieves the content of a blob and save it to a distinct local zip file,...
```
are you okay with it?

Reply:
To be honest, I don't really know what the export does.
If I look at the implementation:
```java
            File exportingDirectory = fileSystem.getFile(configuration.exportDirectory);
            FileUtils.forceMkdir(exportingDirectory);

            String fileURL = generateFileUrl(fileExtension);
            File file = fileSystem.getFile(fileURL);
            FileUtils.copyToFile(blobStore.read(blobId), file);
```
It doesn't seem to create a zip at all, is it?

Reply:
`generateFileUrl(fileExtension)` generates a file url with a `.zip` extension. If the problem is only zip file. I will remove the `zip` word in that sentence